Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.tsx

```diff
@@ -32,6 +32,11 @@ const SubmissionGridRow = ({ submission }: ISubmissionGridRowProps) => {
         isOfficial,
     } = submission;
 
+    const getProblemNumber = useMemo(
+        () => problemName.substring(0, 2).substring(1),
```

Comment:
magic numbers

Reply:
I removed the method and instead added a property OrderBy to the problem, which gives me its number.